[2022-08-24 15:49:07,593] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.carga_arquivo manual__2022-08-24T15:49:06.227873+00:00 [queued]>
[2022-08-24 15:49:07,600] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.carga_arquivo manual__2022-08-24T15:49:06.227873+00:00 [queued]>
[2022-08-24 15:49:07,601] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-24 15:49:07,601] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-24 15:49:07,602] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-24 15:49:07,613] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): carga_arquivo> on 2022-08-24 15:49:06.227873+00:00
[2022-08-24 15:49:07,618] {standard_task_runner.py:52} INFO - Started process 95 to run task
[2022-08-24 15:49:07,621] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'carga_arquivo', 'manual__2022-08-24T15:49:06.227873+00:00', '--job-id', '9739', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmp21kto3oe', '--error-file', '/tmp/tmpvkfi_kr7']
[2022-08-24 15:49:07,622] {standard_task_runner.py:80} INFO - Job 9739: Subtask carga_arquivo
[2022-08-24 15:49:07,678] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.carga_arquivo manual__2022-08-24T15:49:06.227873+00:00 [running]> on host 3f878792eb1e
[2022-08-24 15:49:07,741] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=carga_arquivo
AIRFLOW_CTX_EXECUTION_DATE=2022-08-24T15:49:06.227873+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-24T15:49:06.227873+00:00
[2022-08-24 15:49:12,709] {python.py:173} INFO - Done. Returned value was: None
[2022-08-24 15:49:12,797] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=carga_arquivo, execution_date=20220824T154906, start_date=20220824T154907, end_date=20220824T154912
[2022-08-24 15:49:12,828] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-24 15:49:12,851] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
